appdx-dev-tools.asciidoc

```diff
@@ -2,6 +2,102 @@
 [[dev_tools_frameworks]]
 == Development Tools, Frameworks, pass:[<span class="keep-together">and Libraries</span>]
 
+
+=== Development Tools
+
+[[alchemy]]
+==== Alchemy API
+
+Website: https://alchemyapi.io
+
+Documentation: https://docs.alchemyapi.io/
+
+Alchemy API is a Ethereum infrastructure platform and node provider that maintains nodes as cloud services and executes node requests on your behalf. It provides access to managed Geth and Parity nodes, a set of developer tools that simplify Ethereum development, and tools to monitor and notify you on the status of your transactions.
+
+Without a node infrastructure provider like Alchemy, Ethereum developers need to spin up their own self-hosted nodes, which can cost an average of $86k a year accounting for AWS costs and engineering time. These nodes are time consuming, requiring months of engineering time to develop robust infrastructure, and failures can take up to 3 weeks to recover. On top of that, these nodes can be extremely unreliable - running into issues with CPU spikes, memory leaks, peering failures, corrupted databases, and consistency issues when nodes fall out of sync with the network. 
```

Comment:
Maybe throw in "Spin up their own self hosted nodes that require intensive resources and can cost an average of $86k a year" to clarify its difficult to run locally?

Reply:
I actually made it shorter because of Dphil's suggestion in Slack!